examples/high_level_functions_basic_objects.js

```diff
@@ -0,0 +1,253 @@
+memberEccentricity.RelativeToSection(undefined, [], [], "SECTION_ALIGNMENT_RIGHT_TOP");
+memberEccentricity.RelativeAndAbsolute(undefined, [], [], "SECTION_ALIGNMENT_RIGHT_BOTTOM", 0.005, 0.0015, 0.002);
+member5.Eccentricities(1, 2);
+// Option: member support
+var member6 = new Member();
+member6.Beam(undefined, [37, 38], 1);
+var memberSupport = new MemberSupport();
+memberSupport.Fixed();
+member6.Supports(1);
+// Option: nonlinearity
+var member7 = new Member();
+member7.Beam(undefined, [39, 40], 1);
+var memberNonlinearity = new MemberNonlinearity();
+memberNonlinearity.FailureIfTensionWithSlippage(undefined, [], 0.05);
+member7.Nonlinearity(1);
+// Option: member result intermediate points
+var memberResultIntermediatePoint = new MemberResultIntermediatePoint(undefined, [], "MemberResultIntermediatePoint with uniform distances");
+memberResultIntermediatePoint.UniformDistances(5, ["ord. 1", "ord. 2", "ord. 3", "ord. 4", "ord. 5"]);
+var member8 = new Member();
+member8.Beam(undefined, [41, 42], 1);
+member8.ResultIntermediatePoints(1);
+// Option: End modifications
+var member9 = new Member();
+if (RFEM) {
+	member9.Beam(undefined, [43, 44, 56, 46, 47], 1);
+}
+else {
+	member9.Beam(undefined, [43, 44], 1);
+}
+member9.EndModifications([0.1, 0.2, 0.2], [0.5, 0.2, 0.3]);
+// Option: Deactivate for calculation
+member9.DeactivateForCalculation();
+/*********************************************************************************************/
+
+if (RFEM) {
+	/***************************************** Lines *********************************************/
+	var line = new Line(undefined, [61, 62], "First line with default type");
+	var line2 = new Line();
+	line2.Polyline(undefined, [63, 74, 64, 75, 65]);
+	line2.Arc(undefined, [66, 68], [-10, 7, 0]);
+	line2.Arc(undefined, [69, 70], [-8, 7, 0], [undefined, undefined, 2.5]);	// with α parameter defined
+	line2.Arc(undefined, [71, 72], [-24, 8, 0], undefined, [-29, 7.5, 0]);		// with moved center of arc
+	line2.Circle(undefined, [nodes[83].coordinate_1, nodes[83].coordinate_2, nodes[83].coordinate_3], 2, [1, 0, 0]); // with normal parallel to X-Axis
+	line2.EllipticalArc(undefined, [nodes[84].coordinate_1, nodes[84].coordinate_2, nodes[84].coordinate_3], [nodes[87].coordinate_1, nodes[87].coordinate_2, nodes[87].coordinate_3], [nodes[96].coordinate_1, nodes[96].coordinate_2, nodes[96].coordinate_3]);
+	line2.Ellipse(undefined, [88, 100], [nodes[98].coordinate_1, nodes[98].coordinate_2, nodes[98].coordinate_3]);
+	line2.Parabola(undefined, [81, 93], [nodes[101].coordinate_1, nodes[101].coordinate_2, nodes[101].coordinate_3]);
+	line2.Spline(undefined, [103, 94, 105, 95, 106]);
+	line2.NURBS(undefined, [107, 119], [[nodes[109].coordinate_1, nodes[109].coordinate_2, nodes[109].coordinate_3, undefined], [nodes[120].coordinate_1, nodes[120].coordinate_2, nodes[120].coordinate_3, undefined], [nodes[120].coordinate_1, nodes[120].coordinate_2, nodes[120].coordinate_3, undefined]], 5);
```

Comment:
Duplicity node N120x in control point -> invalid object is created. control points cannot be same